ivanrodri/react-boilerplate/readme.md

```diff
@@ -0,0 +1,76 @@
+
+## Entorno de desarrollo
+
+Para agilizar un poco el desarrollo y evitar cometer los típicos errores tontos cuando desarrollamos en JavaScript (usar variables no declaradas, escribir mal las variables, etc), hemos añadido un linter, que nos va a permitir identificar errores sintácticos directamente en el IDE, esto nos ayudará a agilizar un poco el proceso. Unificar el estilo de código entre desarrolladores, nos puede ayudar a entender mejor el código en algunas ocasiones, para eso, hemos añadido **prettier** que va a ser un formateador de código, una vez que guardemos, formateará el código de la manera que hayamos definido nuestras reglas, de esta manera todos tendremos el mismo estilo de código.
+
+Una de las cosas mas importantes a la hora de desarrollar es tener un entorno de desarrollo **perfectamente configurado**, unas veces puede ser por falta de conocimiento de extensiones o por pereza que no tenemos bien configurado el IDE, por ello, hemos creado una [**imagen docker**](https://hub.docker.com/r/nocountryforgeeks/vscode-js/) con **VS Code** totalmente configurado y listo para trabajar.
+
+El proyecto tiene configurado **HMR** (Hot Module Replacement) que nos permitirá reflejar los cambios en caliente sin perder el estado de la aplicación y con una rpidez inmediata.
+
+## Docker
+
+## Overview
+
+Esta es una primera versión donde se pueden mejorar muchas cosas y en lo que estamos trabajando, para poder ofrecer la máxima flexibilidad y añadiendo nuevas funcionalidades. Tenemos marcado un **Road Map** de mejora y actualización:
+
+- Creación de un provider de temas
+- Creación de un CLI
+    - Generar el proyecto a través del CLI
+    - Opción de crear un proyecto dinámicamente según configuración requerida
+    - Extraer la configuración webpack como un paquete npm y permitir que sea extensible
+- Diferentes opciones de build
+    - GitHub
+    - Gitlab
+    - VSTS
+- Service worker
+    - Permitir usar la aplicación sin conexión.
+    - Cachear las todas las llamadas
+- Notificaciones
+    - Firebase
+    - Azure
+    - Amazon
+- Uso de servicios nativos (GPS, Webcam, Micrófono)
+- Plash screen
+- Soporte para PWA
+- SSR
+
+#
+
+<h2 align="center">Contribuyentes</h2>
+
+<table>
```

Comment:
Use [Markdown tables](https://www.tablesgenerator.com/markdown_tables) instead of this abomination.